test/controllers/groups_controller_test.rb

```diff
@@ -0,0 +1,66 @@
+require "test_helper"
+
+class GroupsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @user = users(:one)
+    @groupsss = groups(:one)
+    @user.groups.build(
```

Comment:
There's no need to use `build` if you're using `.save` right after
This could be done this wasy
```
@groups = @user.groups
               .create!(from_time: @groupsss.from_time,
                        name: @groupsss.name, 
                        to_time: @groupsss.to_time)
```

Reply:
You're right!